File: ModelRelief/Scripts/Api/V1/Interfaces/IModel.ts
// ------------------------------------------------------------------------// 
// ModelRelief                                                             //
//                                                                         //                                                                          
// Copyright (c) <2017-2018> Steve Knipmeyer                               //
// ------------------------------------------------------------------------//
"use strict";

import {ContentType, HttpLibrary, 
    MethodType, ServerEndPoints}        from 'Http'
    import { RequestResponse }                  from 'RequestResponse'

 /**
 *  Common interface for all DTO model types.
 *  @interface
 */
export interface IModel {

    id?: number;
    name?: string;
    description?: string;
}

File: ModelRelief/Features/Dto/IModel.cs
// -----------------------------------------------------------------------
// <copyright file="IModel.cs" company="ModelRelief">
// MIT License (https://github.com/steveknipmeyer/ModelRelief/blob/main/MIT-LICENSE.txt)
// </copyright>
// -----------------------------------------------------------------------

namespace ModelRelief.Dto
{
    /// <summary>
    /// Common interface for all model types.
    /// </summary>
    public interface IModel
    {
        int Id { get; set; }
        string Name { get; set; }
        string Description { get; set; }
    }
}
